src/archivematicaCommon/lib/storageService.py

```diff
@@ -1,16 +1,19 @@
 from __future__ import absolute_import
-import base64
+
 import logging
 import os
 import platform
-import requests
-from requests.auth import AuthBase
-import urllib
 
 from django.conf import settings as django_settings
+import requests
+from requests.auth import AuthBase
+import six.moves.urllib.error
+import six.moves.urllib.parse
+import six.moves.urllib.request
```

Comment:
This feels very Javaish! Another exmaple where we could just import `urllib` via local binding.